Documentation/network/egress-gateway.rst

```diff
@@ -278,6 +278,10 @@ Regardless of which way the egress IP is configured, the user must ensure that
 Cilium is running on the device that has the egress IP assigned to it, by
 setting the ``--devices`` agent option accordingly.
 
+.. warning::
+
+   The ``egressIP`` and ``interface`` properties cannot both be specified in the `egressGateway` spec. Egress Gateway Policies that contain both of these properties will be ignored by Cilium.
```

Comment:
> Egress Gateway Policies that contain both of these properties will be ignored by Cilium.

Can the user expect a warning in the agent logs in that case? If no, should we add one?

Reply:
There is a warning in place, however it comes from the `k8s` subsystem rather than the `egressgateway` subsystem on Cilium 1.14, not sure about 1.15.